src/components/ToneBox/ToneBox.jsx

```diff
@@ -1,44 +1,28 @@
 import React from 'react';
-import { Card, CardHeader } from 'material-ui/Card';
+import { Card } from 'material-ui/Card';
 import classes from './ToneBox.scss';
-import api from 'services';
 
-class ToneBox extends React.Component {
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      toneResult: [],
-    };
-  }
-  componentDidMount() {
-    api.getTone(this.props.conversationID).then(toneAnalysis => this.setState({
-      toneResult: toneAnalysis.map(emotion => ({ text: emotion.tone_name, value: emotion.score })),
-    }));
+const ToneBox = (props) => (
+  <Card className={classes.container}>
+    {props.toneResult.length > 0 ?
+      <div>
+        Watson Analytics
+        <hr />
+        {props.toneResult.map((emotion, i) =>
+          <li key={i}>
+            <p>{emotion.text}: {(emotion.value * 100).toFixed(2)}%</p>
+          </li>
+        )}
+      </div>
+    :
+      <img alt="loading" src="/watson_anim.gif" height="180px" width="180px" />
   }
+  </Card>
+  );
 
-  render() {
-    return (
-      <Card className={classes.container}>
-        {this.state.toneResult.length > 0 ?
-        <div>
-          Watson Analytics
-          <hr />
-          {this.state.toneResult.map((emotion, i) =>
-            <li key={i}>
-              <p>{emotion.text}: {(emotion.value * 100).toFixed(2)}%</p>
-            </li>
-          )}
-        </div>
-        :
-        <img src="/watson_anim.gif" height="180px" width="180px" />
-      }
-      </Card>
-    );
-  }
-}
 ToneBox.propTypes = {
-  conversationID: React.PropTypes.string.isRequired,
+  toneResult: React.PropTypes.array.isRequired,
```

Comment:
For objects / arrays of objects let's define them a bit more. Will be much easier to understand what props are available or expected when we come back to the code (or other people inspect it)

IE:
```javascript
ToneBox.propTypes = {
  toneResult: React.PropTypes.arrayOf(React.PropTypes.shape({
    text: React.PropTypes.string.isRequired,
    value: React.PropTypes.number.isRequired,
})).isRequired,
}
```